frontend/routes/Wifi.jsx

```diff
@@ -13,41 +16,95 @@ export class Component extends React.Component {
       last_refresh: '',
     };
     setInterval(() => {
```

Comment:
It would be a good idea to cleanup the `setInterval` for this one aswell.

Reply:
Taken care in upcoming PR